src/test/groovy/org/prebid/server/functional/model/UidsCookie.groovy

```diff
@@ -1,7 +1,8 @@
-package org.prebid.server.functional.model.request.setuid
+package org.prebid.server.functional.model
```

Comment:
What was the point in moving it up in the hierarchy?

Reply:
Because this class is used for both request and response of the setuid